docs/sdk/src/reference_docs/frame_tokens.rs

```diff
@@ -0,0 +1,129 @@
+//! but instead accepts any dependency which implements the
+//! [`currency::ReservableCurrency`](`frame_support::traits::tokens::currency::ReservableCurrency`)
+//! trait, namely via its [`Config::Currency`](`pallet_preimage::pallet::Config::Currency`)
+//! associated type. This allows [`pallet_preimage`] to support any arbitrary pallet implementing
+//! this trait, without needing any knowledge of what those pallets may be or requiring changes to
+//! support new pallets which may be written in the future.
+//!
+//! Read more about coupling, and the benefits to loose coupling
+//! [here](crate::reference_docs::frame_pallet_coupling).
+//!
+//! ##### *Rust Advanced Tip
+//!
+//! The knowledge that [`pallet_balances`] implements
+//! [`fungible::Inspect`](`frame_support::traits::fungible::Inspect`) is not some arcane knowledge
+//! that you have to know by heart or memorize. One can simply look at the list of the implementors
+//! of any trait in the Rust Doc to find all implementors (e.g.
+//! <https://paritytech.github.io/polkadot-sdk/master/frame/traits/tokens/fungible/trait.Mutate.html#implementors>),
+//! or use the `rust-analyzer` `Implementations` action.
+//!
+//! ## Fungible Token Traits in FRAME
+//!
+//! The [`fungible`](`frame_support::traits::fungible`) crate contains the latest set of FRAME
+//! fungible token traits, and is recommended to use for all new logic requiring a fungible token.
+//! See the crate documentation for more info about these fungible traits.
+//!
+//! [`fungibles`](`frame_support::traits::fungibles`) provides very similar functionality to
+//! [`fungible`](`frame_support::traits::fungible`), except it supports managing multiple tokens.
+//!
+//! You may notice the trait [`Currency`](`frame_support::traits::Currency`) with similar
+//! functionality is also used in the codebase, however this trait is deprecated and existing logic
+//! is in the process of being migrated to [`fungible`](`frame_support::traits::fungible`) ([tracking issue](https://github.com/paritytech/polkadot-sdk/issues/226)).
+//!
+//! ## Fungible Token Trait Implementations in FRAME
+//!
+//! [`pallet_balances`] implements [`fungible`](`frame_support::traits::fungible`), and is the most
+//! commonly used fungible implementation in FRAME. Most of the time, it's used for managing the
+//! native token of the blockchain network it's used in.
+//!
+//! [`pallet_assets`] implements [`fungibles`](`frame_support::traits::fungibles`), and is another
+//! popular fungible token implementation. It supports the creation and management of multiple
+//! assets in a single crate, making it a good choice when a network requires more assets in
+//! addition to its native token.
+//!
+//! ## Non-Fungible Tokens in FRAME
+//!
+//! [`pallet_uniques`] is recommended to use for all NFT use cases in FRAME.
+//! See the crate documentation for more info about this pallet.
+//!
+//! [`pallet_nfts`] is deprecated and should not be used.
```

Comment:
The other way around

Reply:
Omg good catch.. Not sure how I mixed that up 😂